L1Trigger/TrackFindingTracklet/python/Demonstrator_cfi.py

```diff
@@ -3,9 +3,12 @@
 
 TrackTriggerDemonstrator_params = cms.PSet (
 
-  LabelIn  = cms.string( "TrackFindingTrackletProducerDRin"    ), #
+  LabelIn  = cms.string( "TrackFindingTrackletProducerTM"    ), #
```

Comment:
All _cfi.py and cff.py files should have a comment at the top saying what they are for.

Reply:
done